internal/bpf/bpfWrapper.c

```diff
@@ -13,10 +13,13 @@
  * limitations under the License.
  */
 
-#include <bpf/xsk.h>	   // for xsk_setup_xdp_prog, bpf_set_link_xdp_fd
 #include <linux/if_link.h> // for XDP_FLAGS_UPDATE_IF_NOEXIST
 #include <net/if.h>	   // for if_nametoindex
-
+#if __has_include(<xdp/xsk.h>)
+#include <xdp/xsk.h>
+#else
+#include <bpf/xsk.h>
+#endif
 #include "bpfWrapper.h"
 #include "log.h"
 

```

Comment:
The function

`int Load_bpf_send_xsk_map(char *ifname) `

should be renamed to 

`int Load_bpf_xsk_prog(char *ifname)`